testing_requirements.txt

```diff
@@ -1,3 +1,4 @@
+django==1.11
```

Comment:
Good spot that I haven't included this anywhere in the dependencies, but because we [support multiple django versions](https://github.com/DemocracyClub/dc_signup_form/blob/8d52f28461993c42a29214b99fd072b0e9534933/.travis.yml#L9-L11) I think the correct way to specify that we need django is to add `"Django >=1.10,<2.1"` to `install_requires` in `setup.py`. That way we:

* Specify that django is a dependency
* Ensure a compatible version (django 2) will be installed if we set up from scratch
* Use whatever version is already installed if we set up in an existing project running a compatible version

Reply:
While you're editing this file, can you also update it so we're testing with the latest DC Theme: `git+git://github.com/DemocracyClub/dc_base_theme.git@0.3.7` - cheers.

Reply:
Actually, maybe the Django version isn't needed in there at all – I'd missed that this is being installed by Travis in `.travis.yml` already, according to the build matrix. I only needed it for local testing.

I guess it should be added to `setup.py` too, but only for other projects dependency trees, not to help local tests (assuming `pip install .` isn't used)